web/Makefile

```diff
@@ -92,10 +89,24 @@ quarto: poetry pak $(QUARTO_MD_OUTPUTS)
 
 clean:
 	rm -f $(CHANGELOG_OUTPUT)
-	rm -rf .github
 	rm -f $(RFC_OUTPUT)
 	rm -f $(CONTRIBUTING_DIR)/code-of-conduct.md
 	rm -f $(CONTRIBUTING_DIR)/coding-style.md
 	rm -f $(CONTRIBUTING_DIR)/security.md
 	rm -f $(CONTRIBUTING_DIR)/workflow.md
 	rm -rf $(QUARTO_MD_OUTPUTS) $(QUARTO_FILES_OUTPUTS)
+
+
+# A straitforward target to enable `make docker TARGET=bash`
+bash: 
+	@bash
+
+# Run this makefile from within the Quarto Docker image.
+# You can specify another target using the TARGET variable:
+# make docker TARGET=clean
+docker:
+	HOST_UID=$$(id -u) HOST_GID=$$(id -g) docker compose \
+		-f ../docker/quarto/docker-compose.yaml \
+		-f ../docker/quarto/docker-compose.bind.yaml \
+		run quarto \
```

Comment:
There's a quasi standard for making sure that arguments after a standalone double dash are considered positional arguments, so for example `lldb $(which vast) -- --endpoint=:42001 start` would consider `--endpoint=:42001` as a positional argument to lldb rather than an option, which causes it to be passed through to VAST. The same applies to Git, which uses it to disambiguate between "ref or file" and "must be file."

Reply:
well, just tested, and it doesn't work with docker run, so it seems it would rather be DMDV (Docker's Mileage Does Vary) 😄